test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -14,14 +14,48 @@ namespace KozaniManagerTestPackage
     constexpr PCWSTR c_PackageFullName = L"KozaniManager_1.0.0.0_neutral__8wekyb3d8bbwe";
 }
 
+HRESULT RegisterDVCPluginIfNeeded()
+{
+    PCWSTR subkey = L"Software\\Microsoft\\Terminal Server Client\\Default\\AddIns\\KozaniDVC";
+    PCWSTR keyname = L"Name";
+
+    wil::unique_hkey key;
+    LSTATUS result = ::RegOpenKey(HKEY_CURRENT_USER, subkey, &key);
+    if (result == ERROR_SUCCESS)
+    {
+        WEX::Logging::Log::Comment(L"DVC plugin key already exists!");
+        return S_OK;
+    }
+
+    PCWSTR kozaniDvcClientClsid{ L"ed202fc8-ee21-4bf2-9681-23a8e97be9f3" };
+    WCHAR regCommand[MAX_PATH];
```

Comment:
NIT: `...[MAX_PATH]{};`

Always make sure local variables aren't uninitialized.

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

Same comment applies throughout

in this case, grep the code for "];" and you'll find any uninitialized arrays

Reply:
There are exceptions for arrays, otherwise one will not be able to write efficient code, or worse, always write in-efficient code. I understand the risk of uninitialized data but in this case, it is very simple usage and obviously will not cause any uninitialized data access issues. Initializing large array unnecessarily is bad practice. Say, if you have a 10MB buffer waiting to be filled, do you waste the resources to first fill it with 0s?

[Exception](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax:~:text=Exception%20If%20you%20are,be%20treated%20with%20suspicion.)

Exception If you are declaring an object that is just about to be initialized from input, initializing it would cause a double initialization. However, beware that this might leave uninitialized data beyond the input – and that has been a fertile source of errors and security breaches:
constexpr int max = 8 * 1024;
int buf[max];         // OK, but suspicious: uninitialized
f.read(buf, max);
The cost of initializing that array could be significant in some situations. However, such examples do tend to leave uninitialized variables accessible, so they should be treated with suspicion.